hbase_storage/pom.xml

```diff
@@ -27,6 +27,11 @@
             <artifactId>pipeline_api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.splicemachine</groupId>
+            <artifactId>splice_machine</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Why is this needed? In principle I'd say `hbase_storage` shouldn't depend on `splice_machine`, since `splice_machine` should be a higher level module.

Reply:
It's needed for DataValueDescriptor in KeyPrefixProbingFilter.  I've refactored that class so that it's no longer needed.

Reply:
I've added a benchmark test.